mne/io/eyelink/eyelink.py

```diff
@@ -284,6 +287,45 @@ def _find_overlaps(df, max_time=0.05):
     return ovrlp.drop(columns=tmp_cols).reset_index(drop=True)
 
 
+@fill_doc
+def read_eyelink_calibration(
+    fname, screen_size=None, screen_distance=None, screen_resolution=None
+):
+    """Return info on calibrations collected in an eyelink file.
+
+    Parameters
+    ----------
+    fname : str
+        Path to the eyelink file (.asc).
+    screen_size : tuple of shape (2,)
```

Comment:
Could we accept an `array-like` and cast to `tuple` if needed?

Reply:
Sure, that's a good idea.

This information is just stored in the `Calibration` class and displayed for the user via the `__repr__`  method -  but right now there aren't any functions in `mne` that actually do anything with `screen_size` / `screen_resolution`.  So at the moment I'm not sure if it's even necessary to cast them as `tuple`'s?

We plan to write a function named something like `px_to_degrees`, that would use these values, so maybe we should decide then on whether to automatically caste `screen_size`/`screen_resolution` as `tuple`'s?